rajawali/src/main/java/c/org/rajawali3d/engine/RenderView.java

```diff
@@ -0,0 +1,66 @@
+package c.org.rajawali3d.engine;
+
+import android.graphics.Rect;
+import android.support.annotation.Nullable;
+import c.org.rajawali3d.annotations.GLThread;
+
+/**
+ * Interface to be implemented by views that are to be managed by the Engine. A RenderView represents
+ * a viewport on the render surface
+ *
+ * @author Jared Woolston (Jared.Woolston@gmail.com)
+ */
+public interface RenderView {
```

Comment:
I'm thinking it makes sense to have `RenderView` implement `SurfaceCallback` or otherwise be notified of the surface size changed (such as through a method independent of `SurfaceCallback` that is part of the `RenderView` interface). This would allow `RenderView` to have enough information to use relative coordinates for its size and automatically correct its viewport settings for changes.